ssha/ssh.py

```diff
@@ -35,11 +33,13 @@ def connect(instance, bastion):
         command += ['-o', 'UserKnownHostsFile={}'.format(user_known_hosts_file)]
 
     if bastion:
-        config.add('bastion.address', _get_address(bastion))
+        instance_ip = get_ip(bastion, False)
```

Comment:
I'd prefer:

```python
instance_ip = get_ip(bastion, connect_through_bastion=False)
```

And similar for the other use of `get_ip()` because that `False` by itself is mysterious.